test/scripts/initialize_eks_cluster.bash

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+LD_FLAGS=$1
```

Comment:
The image is built using makefile `container` target, so this isn't required. Can you confirm and remove this?

Reply:
Will remove